TicketBookingAPI/TicketBooking.WebAPI/Controllers/ConcertController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Azure;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Text.Json.Nodes;
+using TicketBooking.Application.Features.Concerts.Commands.CreateConcert;
+using TicketBooking.Application.Features.Concerts.Commands.DeleteConcert;
+using TicketBooking.Application.Features.Concerts.Commands.UpdateConcert;
+using TicketBooking.Application.Features.Concerts.Queries.GetConcert;
+using TicketBooking.Application.Features.Concerts.Queries.GetConcertList;
+using TicketBooking.Domain;
+
+using Microsoft.AspNetCore.Identity;
+using System.Security.Claims;
+
+namespace TicketBooking.WebAPI.Controllers
+{
+    [Route("api/[controller]s")]
+    public class ConcertController : BaseController
+    {
+        [HttpGet]
+        [AllowAnonymous]
+        public async Task<ActionResult<ConcertListVm>> GetAll(string? containsInName = "", string? concertType = "")
+        {
+            var query = new GetConcertListQuery() { ContainsInName = containsInName, ConcertType = concertType };
+            var vm = await Mediator.Send(query);
+            return Ok(vm);
+        }
+
+        [HttpGet("{id}")]
+        [AllowAnonymous]
+        public async Task<ActionResult<Concert>> Get(Guid id)
+        {
+            var query = new GetConcertQuery() { Id = id };
+            var vm = await Mediator.Send(query);
+            return Ok(vm);
+        }
+        
+        [HttpPost]
+        [Authorize(Roles = "Admin")]
+        public async Task<ActionResult<Guid>> Create([FromBody] JsonObject jsonObj)
```

Comment:
as alternative you can create `createConcertViewModel` with necessary fields, set as parameters instead `JsonObject` and asp.net automatic parse all fields by name